### PR TITLE
Create API definition for time entries

### PR DESCRIPTION
**Pull request by EliuX**
Creates the API definition for Time Entries, which are those created every time a user starts a new activity.

### CODE REVIEW

File: time_tracker_api/activities/activities_api.py [hunk 1]
```diff
@@ -4,28 +4,72 @@
...
+        required=True,
+        title='Name',
+        max_length=50,
+        description='Canonical name of the activity',
+    ),
+    'description': fields.String(
+        title='Description',
+        description='Comments about the activity',
+    ),
+
```
Comment (Angeluz-07) [inline time_tracker_api/activities/activities_api.py]: You have a space here.

File: time_tracker_api/time_entries/time_entry_api.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
+from flask_restplus import fields, Resource, Namespace
+
+ns = Namespace('time_entries', description='API for time entries')
```
Comment (Angeluz-07) [inline time_tracker_api/time_entries/time_entry_api.py]: You may want to change `time_entries` to `time-entries` as recommendation from https://restfulapi.net/resource-naming/

File: time_tracker_api/activities/activities_api.py (hunk 1 above)
Comment (EliuX) [reply time_tracker_api/activities/activities_api.py]: Refresh it, please :)

File: time_tracker_api/time_entries/time_entry_api.py (hunk 2 above)
Comment (EliuX) [reply time_tracker_api/time_entries/time_entry_api.py]: True, my bad.

**Pull request merged**